jemalloc/plan.sh

```diff
@@ -0,0 +1,33 @@
+pkg_name=jemalloc
+pkg_description="jemalloc"
+pkg_upstream_url=http://www.canonware.com/jemalloc/
+pkg_origin=someara
+pkg_version=4.2.1
+pkg_maintainer="Sean OMeara <sean@sean.io>"
+pkg_license=('BSD derived')
```

Comment:
and we probably should copy `COPYING` as `LICENSE` inside the `.hart` package.  https://github.com/jemalloc/jemalloc/blob/master/COPYING